rwengine/src/objects/VehicleObject.cpp

```diff
@@ -262,14 +262,23 @@ void VehicleObject::tickPhysics(float dt) {
             physVehicle->setBrake(brakeReal * brakeF, w);
 
             if (wi.m_bIsFrontWheel) {
-                float sign = std::signbit(steerAngle) ? -1.f : 1.f;
-                physVehicle->setSteeringValue(
-                    std::min(glm::radians(info->handling.steeringLock),
-                             std::abs(steerAngle)) *
-                        sign,
-                    w);
-                // physVehicle->setSteeringValue(std::min(3.141f/2.f,
-                // std::abs(steerAngle)) * sign, w);
+                float currentVal = physVehicle->getSteeringValue(w);
+                float currentSign = std::signbit(currentVal) ? -1.f : 1.f;
```

Comment:
`std::copysign(1.0f, currentVal)` may be handy here.

Reply:
Good suggestion; nit.